udata/migrations/2015-11-23-set-uuid-as-community-resources-id.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Ensure all community resources have an UUID string as ID
+ */
+
+/**
+ * Generate a rfc4122 version 4 UUID
+ */
+function uuid() {
```

Comment:
This migrations drived me crazy between the lack of nantive uuid support and `replace()` that only replace the first occurence !
I want a python shell for Mongo !!! :trollface: